rune/dist/centos/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: rpm release-tarball
+
+RPMBUILD_DIR := $(shell mktemp -u /tmp/rpmbuild.XXXX)
+INCLAVARE_CONTAINERS_VERSION := $(shell cat ../../../VERSION)
+RELEASE_TARBALL_URL := https://github.com/alibaba/inclavare-containers/archive/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
+RELEASE_TARBALL_FILE := $(RPMBUILD_DIR)/SOURCES/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
+RELEASE_TARBALL_EXIST := $(shell if [ -f $(RELEASE_TARBALL_FILE) ]; then echo "exist"; else echo "notexist"; fi;)
+
+release-tarball:
+ifneq ($(RELEASE_TARBALL_EXIST), exist)
+	@mkdir -p $(RPMBUILD_DIR)/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+	@echo "%_topdir $(RPMBUILD_DIR)" >> ~/.rpmmacros
```

Comment:
Running multi-time would get lots of duplicated "%_topdir $(RPMBUILD_DIR)". Use > enough.